unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/CollapsableSortedFriendEntryList.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using DCL.Helpers;
+using UnityEngine;
+
+public class CollapsableSortedFriendEntryList : MonoBehaviour
```

Comment:
As a collpasable list is something that is surely going to be used in future UI implementations, it could have been a good opportunity for creating a new UI Component (for example called `CollapsableContainer`) in our common library to be used here and wherever we want.